README.md

```diff
@@ -169,12 +169,12 @@ Don't panic. You can call any method with help of `SendRequestAsync` function. F
   //Create request 
   var req = new TLRequestSetTyping()
   {
-    action = new TLSendMessageTypingAction(),
-    peer = peer
+    Action = new TLSendMessageTypingAction(),
+    Peer = new TLInputPeerUser() { UserId = user.Id }
   };
 
   //run request, and deserialize response to Boolean
-  return await SendRequestAsync<Boolean>(req);
+  return await client.SendRequestAsync<Boolean>(req);
```

Comment:
mmm, but `client` variable is not declared anywhere

Reply:
well you can't call SendRequestAsync without it's client, client is defined at the top of README file

Reply:
good point!